umbraco/Umbraco-CMS
# Make dashboards support deep linking

This PR makes dashboards support deep linking. It adds the alias as a query param when switching between dashboards and loads the correct dashboard on page load.

<img width="1057" alt="Screenshot 2021-05-18 at 14 25 25" src="https://user-images.githubusercontent.com/6078361/118650880-39337700-b7e5-11eb-9d83-8f7a93382d3a.png">

**How to test:**
* Make sure you can change dashboards in each section.
* Make sure the query params get updated with the dashboard alias
* Make sure you can link directly to a dashboard
* Make sure the first dashboard is active if you don't have a dashboard query param or type in a random alias
